kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h

```diff
@@ -990,8 +990,11 @@ class ResidualBasedNewtonRaphsonStrategy
             EchoInfo(iteration_number);
 
             // Updating the results stored in the database
+            TSparseSpace::InplaceMult(rDx, 1.0);
```

Comment:
remove changes in the `Newton_raphson` :) I know that it was only to solve that case...

Reply:
Done!